pages/deleteUserPage.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page } from "@playwright/test";
+import { BasePage } from "./basePage";
```

Comment:
as we already have BasePage and extends it(it means we already have constructor), we can remove constructor and initialize variables(locators) at the top of our class without the constructor. 
Moreover, using constructor we will define all variables during creating of the new object(page), but it's not so cool, it's better to use only that locators which we really need for the certain test
Try remove it and check if everything work.